server/src/main/java/com/hedera/block/server/persistence/storage/write/BlockWriter.java

```diff
@@ -30,10 +30,10 @@ public interface BlockWriter<V> {
     /**
      * Write the block item to storage.
      *
-     * @param blockItem the block item to write to storage.
+     * @param toWrite the block item to write to storage.
      * @return an optional containing the block item written to storage if the block item was a
      *     block proof signaling the end of the block, an empty optional otherwise.
      * @throws IOException when failing to write the block item to storage.
      */
-    Optional<V> write(@NonNull final V blockItem) throws IOException;
+    Optional<V> write(@NonNull final V toWrite) throws IOException;
```

Comment:
nit: `toWrite` feels a bit short.  `itemToWrite` reads better.

Reply:
It is not  necessary an item, sometimes it could be a `Block` (not present today) or as it is today `List<BlockItem>` which is plural and still does not match the semantics. I rely on the parameter's type to supplement the semantics for what we are writing, so whatever the type will be, it will always be followed by `toWrite`.

Reply:
`itemToWrite` is a generic term; it could be anything.  The goal of the rename is to have a complete noun, and not a half-word that's confusing.
`valueToWrite` works just as well.

Reply:
I see... I did not read it that way and thought you were refering to a block item. Said that way, makes perfect sense. Thanks.